providers/mysql/tests/unit/mysql/hooks/test_mysql.py

```diff
@@ -86,12 +86,89 @@ def test_dummy_connection_setter(self, mock_connect):
+        [
+            (
+                {
+                    "login": "login",
+                    "password": "password",
+                    "host": "host",
+                    "schema": "schema",
+                    "port": None,
+                    "extra": json.dumps({"charset": "utf-8"}),
+                },
+                "mysql://login:password@host/schema?charset=utf-8",
+            ),
+            (
+                {
+                    "login": "user@domain",
+                    "password": "pass/word!",
+                    "host": "host",
+                    "schema": "schema",
+                    "port": None,
+                    "extra": json.dumps({"charset": "utf-8"}),
+                },
+                "mysql://user%40domain:pass%2Fword%21@host/schema?charset=utf-8",
+            ),
+            (
+                {
+                    "login": "user@domain",
+                    "password": "password",
+                    "host": "host",
+                    "schema": "schema",
+                    "port": None,
+                    "extra": json.dumps({"client": "mysql-connector-python"}),
+                },
+                "mysql+mysqlconnector://user%40domain:password@host/schema",
+            ),
+            (
+                {
+                    "login": "user@domain",
+                    "password": "password",
+                    "host": "host",
+                    "schema": "schema",
+                    "port": 3307,
+                    "extra": json.dumps({"client": "mysql-connector-python"}),
+                },
+                "mysql+mysqlconnector://user%40domain:password@host:3307/schema",
+            ),
+            (
+                {
+                    "login": "user@domain",
+                    "password": "password",
+                    "host": "host",
+                    "schema": "db/name",
+                    "port": 3307,
+                    "extra": json.dumps({"client": "mysql-connector-python"}),
+                },
+                "mysql+mysqlconnector://user%40domain:password@host:3307/db%2Fname",
+            ),
+            (
+                {
+                    "login": "user@domain",
+                    "password": "password",
+                    "host": "host",
+                    "schema": "schema",
+                    "port": 3307,
+                    "extra": json.dumps(
+                        {
+                            "client": "mysql-connector-python",
+                            "ssl_ca": "/path/to/ca",
+                            "ssl_cert": "/path/to/cert with space",
+                        }
+                    ),
+                },
+                "mysql+mysqlconnector://user%40domain:password@host:3307/schema?ssl_ca=%2Fpath%2Fto%2Fca&ssl_cert=%2Fpath%2Fto%2Fcert+with+space",
+            ),
+        ],
+    )
+    def test_get_uri_parametrized(self, mock_connect, connection_params, expected_uri):
```

Comment:
```suggestion
    def test_get_uri(self, mock_connect, connection_params, expected_uri):
```

Leave it as "test_get_uri" is fine, we don't need to mention "parameterize" in method name.

Reply:
Thanks, I've updated!